src/library/client.lua

```diff
@@ -13,6 +13,88 @@ These functions don't accept any arguments, and should always return a boolean.
 ]] --
 local client = {}
 
+function to_human_string(feature)
```

Comment:
Should this (and the ones below) be a `local` functions?

Reply:
Ah yes. Thanks for reminding me!